src/System.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -292,5 +292,19 @@ public static DataTable RunQuery(string connectionString, string sql)
             }
             return result;
         }
+
+        /// <summary>
+        /// This method creates a new SqlConnection object and uses the SqlConnection.DataSource 
+        /// property to return the SQL Server Instance name.
+        /// </summary>
+        /// <param name="connectionString"></param>
+        /// <returns> Returns SQL Server name </returns>
+        public static string GetDataSource(string connectionString)
```

Comment:
Can't this be done using SqlConnectionStringBuilder?

Reply:
Good suggestion, yes it can be done, removed method and replaced method call with the code below instead. 
`new SqlConnectionStringBuilder((DataTestUtility.TcpConnStr)).DataSource`